service/frontend/adminHandler.go

```diff
@@ -204,7 +205,7 @@ func (adh *AdminHandler) DescribeWorkflowExecution(ctx context.Context, request
 	namespaceID, err := adh.GetNamespaceCache().GetNamespaceID(request.GetNamespace())
 
 	shardID := common.WorkflowIDToHistoryShard(namespaceID, request.Execution.WorkflowId, adh.numberOfHistoryShards)
-	shardIDstr := string(shardID)
+	shardIDstr := convert.IntToString(shardID)
 	shardIDForOutput := strconv.Itoa(shardID)
```

Comment:
This `shardIDForOutput` can be safely removed now and replaced with `shardIDstr`.